universal-login-sdk/test/e2e/BalanceObserver.ts

```diff
@@ -34,7 +33,6 @@ describe('E2E: BalanceObserver', () => {
   it('1 subscription - balance changed', async () => {
     const callback = sinon.spy();
 
-    await sdk.fetchBalanceObserver(ensName);
     const unsubscribe = await sdk.subscribeToBalances(ensName, callback);
```

Comment:
In general, I don't feel we need e2e tests for Domain services. 
I would propose unit tests for them + integration tests for the integration layer and e2e for the sdk.